frontend/src/scenes/actions/ActionEdit.tsx

```diff
@@ -136,20 +135,23 @@ export function ActionEdit({ action: loadedAction, id }: ActionEditLogicProps):
                                 </LemonButton>
                             ) : null}
                             {id ? deleteButton() : cancelButton()}
-                            <LemonButton
-                                data-attr="save-action-button"
-                                type="primary"
-                                htmlType="submit"
-                                loading={actionLoading}
-                                onClick={submitAction}
-                            >
-                                Save
-                            </LemonButton>
+                            {actionChanged || !id ? (
+                                <LemonButton
+                                    data-attr="save-action-button"
+                                    type="primary"
+                                    htmlType="submit"
+                                    loading={actionLoading}
+                                    onClick={submitAction}
+                                    disabledReason={!actionChanged && !id ? 'No changes to save' : undefined}
+                                >
+                                    Save
+                                </LemonButton>
+                            ) : null}
                         </>
                     }
                 />
 
-                <div>
+                <div className="@container">
```

Comment:
What do the `@` symbols do in these cases?

Reply:
This is the tailwind container queries format.